src/pages/Home.vue

```diff
@@ -1,29 +1,34 @@
 <template>
-  <div class="container mx-auto px-4 lg:px-0 pt-10 md:pt-12">
+  <div class="invest-container mx-auto pt-10 md:pt-12">
```

Comment:
I really think we should stick to the tailwind `container` class here as we're using it on the pool page too. Keeps everything consistent.

Reply:
The Tailwind container class has a bunch of max-widths, like this

![Screen Shot 2021-07-13 at 10 46 09 AM](https://user-images.githubusercontent.com/1121139/125500539-979398a2-a994-4acf-9d2f-91404eaf2803.png)

This ends up creating a lot of weird layouts like this:

![Screen Shot 2021-07-13 at 10 45 49 AM](https://user-images.githubusercontent.com/1121139/125500627-05950b5b-cc2a-42c5-b59c-5991d2842cd7.png)

This is a really bad use of screen real estate, especially when we have a horizontally scrolling table which really should extend to the right edge of the screen. We're much better off having true responsive, fluid layouts with side padding which use the full screen space instead of having arbitrary max-widths.

Reply:
Ok, I don't think this layout is necessarily to do with the container class, let me play around with this too.

Reply:
This is a duplicate discussion about containers, will resolve in favour of above thread.